src/test/testMergeSchemas.ts

```diff
@@ -125,7 +126,7 @@ if (process.env.GRAPHQL_VERSION !== '^0.11') {
   });
 }
 
-let linkSchema = `
+let linkSchema = gql`
```

Comment:
By switching to use `gql` here, we're no longer testing string based schema's. I think there's value in testing both approaches. Could you maybe leave `linkSchema` as a string here, then add a simple test somewhere just to make sure `DocumntNode` based schemas passed into `mergeSchemas` work?

Reply:
Awesome! I'm glad to hear it @hwillson 

I am certainly happy to add an additional test specifically for testing `DocumentNode` schemas. As currently written there is a combination of `String` and `DocumentNode` schemas in the specific test.

I did feel it was important to test merging *both* at the same time. Will this suffice or would you still like me add a new test?

https://github.com/apollographql/graphql-tools/blob/c44842f265e8bbd07e9d5b5ab8d7314642c73066/src/test/testMergeSchemas.ts#L366-L378